City-of-Helsinki/drupal-helfi-sote
# UHF-2526 Translations

How to test: 
- Checkout this branch 
- Run `composer require drupal/helfi_platform_config:dev-UHF-2526_translations`
- Run `make new`
- Run `make drush-cex`
- Check that there are only few overridden configuration files in `conf/cmi/languages` folder
